UnitTests/SequenceExtensionsFixture.cs

```diff
@@ -21,6 +22,29 @@ public void PerformSequence()
 		}
 
 		[Fact]
+		public void PerformSequenceAsync()
+		{
+			var mock = new Mock<IFoo>();
+
+			mock.SetupSequence(x => x.DoAsync())
+				.ReturnsAsync(2)
+				.ReturnsAsync(3)
+				.ThrowsAsync(new InvalidOperationException());
+
+			Assert.Equal(2, mock.Object.DoAsync().Result);
```

Comment:
Maybe it's time to drop the ball on .NET < 4.5? Since it isn't even a
aupported version anymore, is it?

On Wed, May 18, 2016 at 1:40 PM Alexander Batishchev <
notifications@github.com> wrote:

> In UnitTests/SequenceExtensionsFixture.cs
> <https://github.com/moq/moq4/pull/261#discussion_r63738514>:
>
> > @@ -21,6 +22,29 @@ public void PerformSequence()
> >  		}
> >
> >  		[Fact]
> > +		public void PerformSequenceAsync()
> > +		{
> > +			var mock = new Mock<IFoo>();
> > +
> > +			mock.SetupSequence(x => x.DoAsync())
> > +				.ReturnsAsync(2)
> > +				.ReturnsAsync(3)
> > +				.ThrowsAsync(new InvalidOperationException());
> > +
> > +			Assert.Equal(2, mock.Object.DoAsync().Result);
>
> Because of .NET 4.0 I can't properly await DoAsync() here and below.
>
> —
> You are receiving this because you modified the open/close state.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/moq/moq4/pull/261/files/fe053cec0eb2592edaa4730b5aa8ca6e9acaedc0#r63738514>
>